fast64_internal/oot/file_settings.py

```diff
@@ -24,9 +24,9 @@ def draw(self, context):
             prop_split(col, context.scene.fast64.oot, "oot_version_custom", "Custom Version")
 
         col.prop(context.scene.fast64.oot, "headerTabAffectsVisibility")
-        col.prop(context.scene.fast64.oot, "hackerFeaturesEnabled")
+        col.prop(context.scene.fast64.oot, "feature_set")
 
-        if not context.scene.fast64.oot.hackerFeaturesEnabled:
+        if bpy.context.scene.fast64.oot.feature_set == "decomp":
```

Comment:
```suggestion
        if context.scene.fast64.oot.feature_set == "decomp":
```
no need to use bpy there (it's generating an error actually, and we don't need to import bpy since we already have access to context)